UnitTests/BraintreePayPalTests/BTPayPalDriver_Tests.swift

```diff
@@ -344,6 +452,31 @@ class BTPayPalDriver_Checkout_Tests: XCTestCase {
 
     // MARK: - Tokenization
 
+    func testHandleBrowserSwitchReturn_whenBrowserSwitchSucceeds_sendsCorrectParameters() {
```

Comment:
It looks like this test belongs above the Tokenization mark

Reply:
Ah, looks like the test name just isn't very clear. It's testing that we send the correct params to the tokenization endpoint after the browser switch returns successfully. I'll update the test name to clarify that it's related to tokenization.